### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Theophileaseh**
Desktop version of portfolio created

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -279,34 +296,333 @@ section .circle2 {
...
+  line-height: 28px;
+  color: #ebebff;
+  text-decoration: none;
+}
+
+.desktop-footer {
+  display: none;
+}
+
+@media screen and (min-width: 769px) {
```
Comment (Lordkaito) [inline style.css]: - [ ] Good job implementing the media queries for the desktop version, however, the breakpoint for it should be between 992px and 1100px which are the most common screen sizes. So please, change the breakpoint to one in between those ones, and don't forget to apply further changes if needed due to the width change.

File: style.css [hunk 2]
```diff
@@ -279,34 +296,333 @@ section .circle2 {
...
+.footer .email {
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-decoration: none;
+}
+
+.desktop-footer {
+  display: none;
+}
```
Comment (Lordkaito) [inline style.css:430]: - [ ] Your footer is not being shown on the desktop version due to this rule in your CSS. Remember that, if you do a `display: none` outside of a media query, you will need to do `display: block` (or inline or any you need) in order to show your element once you reach the breakpoint. Kindly fix this issue.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Theophileaseh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] In your mobile version, there is some horizontal scrolling, so please check the sizes of your elements and try to fix this issue to keep your project as professional as possible.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (medaminedev66) [reply style.css:430]: - [ ] You have hidden the desktop footer in the mobile version. but the footer is still hidden in the desktop version. I would recommend adding `.desktop-footer {
  display: block;
}` inside the media query so your footer is shown on the desktop version.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Theophileaseh 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Please ensure that your project doesn't have Linters errors. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Check this link for more information [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software)).

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
# STATUS: APPROVED :tada:  :confetti_ball: :fireworks: 
Hi @Theophileaseh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**To Highlight** :1st_place_medal: 
- An excellent job on implementing the requested changes:heavy_check_mark:  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**